apps/web/lib/getBusyTimes.ts

```diff
@@ -19,24 +19,13 @@ async function getBusyTimes(params: {
   const busyTimes: EventBusyDate[] = await prisma.booking
     .findMany({
       where: {
-        AND: [
-          {
-            userId,
-            eventTypeId,
-            startTime: { gte: new Date(startTime) },
-            endTime: { lte: new Date(endTime) },
-          },
-          {
-            OR: [
-              {
-                status: "ACCEPTED",
-              },
-              {
-                status: "PENDING",
-              },
-            ],
-          },
-        ],
+        userId,
+        eventTypeId,
+        startTime: { gte: new Date(startTime) },
+        endTime: { lte: new Date(endTime) },
+        status: {
+          in: [BookingStatus.ACCEPTED],
```

Comment:
This will prevent unconfirmed bookings to block availability as https://github.com/calcom/cal.com/issues/2550